---
permalink: /
title: "About Me"
excerpt: "Diana Lin, MSc, Bioinformatician, Vancouver, BC"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

<meta property="og:title" content="Diana Lin">
<meta property="og:type" content="website" />
<meta property="og:description" content="Diana Lin, MSc, Bioinformatician">
<meta property="og:image" content="/images/dlin.jpg">
<meta property="og:url" content="https://dy-lin.github.io">
<meta name="twitter:card" content="summary_large_image">

My name is Diana Lin, and I’m currently a research programmer and data scientist at BC Cancer. I recently graduateed from the Bioinformatics Graduate program at the University of British Columbia (UBC) in Vancouver, BC. In October 2021, I finished my Master of Science in Bioinformatics. In 2018, I completed my Bachelor of Science in Physiology at McGill University. In between, I held an 8-month internship at the Bioinformatics Technology Lab.

### MSc Thesis
For my Master’s thesis, I developed and executed a scalable bioinformatics-based discovery platform to conduct data mining for anti-microbial peptide (AMP) sequences in publicly available genomic resources. These genomic resources include various tissue RNA-seq reads from a broad selection of amphibians (frogs and toads) and insects (ants, bees, and wasps). Discovered AMP sequences were synthesized and tested for bioactivity against various pathogens and suitability for drug development. The pipeline, rAMPage, can be found [here](https://github.com/bcgsc/rAMPage).

rAMPage is published in _[Antibiotics](https://doi.org/10.3390/antibiotics11070952):_

> Lin, D., Sutherland, D., Aninta, S.I., Louie, N., Nip, K.M., Li, C., Yanai, A., Coombe, L., Warren, R.L., Helbing, C.C., Hoang, L.M.N., & Birol, I. (2022). &quot;Mining Amphibian and Insect Transcriptomes for Antimicrobial Peptide Sequences with rAMPage.&quot; <i>Antibiotics</i> 11(7). https://doi.org/10.3390/antibiotics11070952

<center><img src='/images/rAMPage.png' width='50%'></center>

{::comment}
During the internship, I worked on the AMP project, where I usedg homology-based search methods to datamine for AMP sequences in the spruce genomes of White spruce, Sitka spruce, and Interior spruce (and eventually Engelmann spruce). My focus was on a certain class of AMPs: defensins. I also analyzed differentially expressed genes in the Sitka and Interior spruce transcriptomes, in particular, looking for AMP sequences and associated sequences in its activation pathway.

In addition to the AMP project, I learned how to use bioinformatics tools in genome assembly and annotation, as a part of the Spruce-Up project. Using these skills, I assembled and annotated the Engelmann spruce chloroplast genome, the white pine weevil mitochondrial genome, and annotated the White spruce chloroplast genome.
{:/comment}
